app/models/accounting/transaction.rb

```diff
@@ -56,7 +56,9 @@ class Accounting::Transaction < ActiveRecord::Base
   validates :loan_transaction_type_value, :txn_date, :accounting_account_id, presence: true
   validates :amount, presence: true, unless: :uninitialized_interest?
 
-  delegate :division, to: :project
+  # since we have a new route that has to do with loans
+  # transactions index breaks since not all transactions have loans
```

Comment:
It's an open question to me if it makes sense to have transactions without loans. Do you have any insight on this? If possible, I'd rather add a not-null constraint to transactions.project_id.